src/www/app/tunnel.ts

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {ShadowsocksConfig} from './config';
+import {Config as ShadowsocksConfig} from 'ShadowsocksConfig';
```

Comment:
Let's define a `ShadowsocksSessionConfig` here in this file. It should only contain what we use. It shouldn't have the name.
The `start` method should take that.